librz/arch/types/types-atomic.sdb.txt

```diff
@@ -51,16 +125,56 @@ type.unsigned short=w
 type.unsigned short.size=16
 type.unsigned short.typeclass=Unsigned Integral
 
+unsigned short int=type
+type.unsigned short int=w
+type.unsigned short int.size=16
+type.unsigned short int.typeclass=Unsigned Integral
+
+short unsigned int=type
+type.short unsigned int=w
+type.short unsigned int.size=16
+type.short unsigned int.typeclass=Unsigned Integral
+
 unsigned long=type
 type.unsigned long=d
 type.unsigned long.size=32
 type.unsigned long.typeclass=Unsigned Integral
 
+long unsigned=type
+type.long unsigned=d
+type.long unsigned.size=32
```

Comment:
These atomics can be wrong on other archs.

Reply:
We need this knowledge base.
But besides that, it can be changed via the console, right?
Maybe add a warning when using a certain command?

Reply:
Good point. Let me move them to the corresponding bitness databases.